src/Compilers/CSharp/Portable/Binder/InContainerBinder.cs

```diff
@@ -72,9 +72,10 @@ internal override bool SupportsExtensions
             get { return true; }
         }
 
+#nullable enable
         internal override void GetCandidateExtensionMethods(
             ArrayBuilder<MethodSymbol> methods,
-            string name,
+            string? name,
```

Comment:
>name [](http://example.com/codeflow?start=20&length=4)

What is the scenario when we don't have a name for an extension method?

Reply:
`LookupSymbols` with `includeReducedExtensionMethods = true` and `name = null`.